modules/assim.sequential/R/sda.enkf.R

```diff
@@ -57,17 +57,19 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
   my.read_restart  <- paste0("read_restart.", model)
   my.write_restart <- paste0("write_restart.", model)
   my.split_inputs  <- paste0("split_inputs.", model)
+  # models that don't need split_inputs OR some flag passed via settings
+  # or should there be a split_inputs.ED2 doing nothing?
+  nosi <- c("ED2")
+  no_split <- model %in% nosi
```

Comment:
hack to be fixed. If necessary a split_inputs.ED2 that does nothing is a better solution than hard coding this. The other alternative is to put this info in the model registry file in the model package.